src/components/TutorialHeader/TutorialHeader.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Header,
+  HeaderName,
+  HeaderNavigation,
+  HeaderMenuItem,
+  HeaderGlobalBar,
+  HeaderGlobalAction,
+  SkipToContent,
+} from 'carbon-components-react/lib/components/UIShell';
+
+import Notification20 from '@carbon/icons-react/lib/notification/20';
+import UserAvatar20 from '@carbon/icons-react/lib/user--avatar/20';
+import AppSwitcher20 from '@carbon/icons-react/lib/app-switcher/20';
+
+const TutorialHeader = () => (
+  <Header arial-label="Carbon Tutorial">
```

Comment:
Sadly Carbon doesn't use TS types/prop types so it can't tell you that `arial-label` isn't valid. :(